src/main/java/com/conveyal/r5/analyst/cluster/GridResultAssembler.java

```diff
@@ -79,19 +79,19 @@
 
     public int nTotal;
 
-    private final String outputBucket;
+    public final String outputBucket;
 
     /** Number of iterations for this grid request */
     private int nIterations;
 
-    public GridResultAssembler (GridRequest request, String outputBucket) {
+    public GridResultAssembler (AnalysisRequest request, String outputBucket) {
```

Comment:
There are no longer subclasses, and instinctively I then think there should be a check that the parameter is the right kind of AnalysisRequest. But that amounts to writing our own type checking, rather than just using the Java type system. We've still got manually coded polymorphism in the sense that there are lots of `if (request.isHighPriority())`, and fields in the requests that are only used in one case or the other. I'm thinking we should still have two subclasses of request.